pinot-core/src/main/java/org/apache/pinot/core/segment/processing/aggregator/PercentileKLLSketchAggregator.java

```diff
@@ -0,0 +1,70 @@
+/**
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+package org.apache.pinot.core.segment.processing.aggregator;
+
+import java.util.Map;
+import org.apache.datasketches.common.SketchesArgumentException;
+import org.apache.datasketches.kll.KllDoublesSketch;
+import org.apache.pinot.core.common.ObjectSerDeUtils;
+import org.apache.pinot.segment.spi.Constants;
+import org.apache.pinot.spi.utils.CommonConstants;
+
+
+/**
+ * Class to merge KLL doubles sketch for minion merge/rollup tasks.
+ */
+public class PercentileKLLSketchAggregator implements ValueAggregator {
+
+  protected static final int DEFAULT_K_VALUE = 200;
```

Comment:
Can be removed here as you've added `DEFAULT_KLL_SKETCH_K` to the Helix common constants.